cmd/aro/gateway.go

```diff
@@ -90,14 +90,19 @@ func gateway(ctx context.Context, log *logrus.Entry) error {
 		return err
 	}
 
+	healthl, err := utilnet.Listen("tcp", ":8081", pkggateway.SocketSize)
```

Comment:
Should we have an `EXPOSE` statement for this port for our docker image?

Reply:
I don't think it is necessary, since we have the `-p` directive on the command line. `EXPOSE` if for when you want to have your port available inside a docker network, it is redundant when you want to expose the port to the outside world